docs/articles/documentation/test-api/selecting-page-elements/selector-options.md

```diff
@@ -2,25 +2,54 @@
 layout: docs
 title: Selector Options
 permalink: /documentation/test-api/selecting-page-elements/selector-options.html
-checked: true
+checked: false
 ---
 # Selector Options
 
-You can pass the following options to the [Selector constructor](selectors.md#creating-selectors) and the
-[t.select](selectors.md#one-time-selection) function.
+You can pass the following options to the [Selector constructor](selectors.md#creating-selectors).
 
-* [options.dependencies](#optionsdependencies)
 * [options.boundTestRun](#optionsboundtestrun)
-* [options.text](#optionstext)
-* [options.index](#optionsindex)
 * [options.timeout](#optionstimeout)
 * [options.visibilityCheck](#optionsvisibilitycheck)
+* (***deprecated***) [options.dependencies](#optionsdependencies)
+*  (***deprecated***) [options.text](#optionstext)
+*  (***deprecated***) [options.index](#optionsindex)
+
 
 You can also overwrite the options you have specified before.
 
 * [Overwriting Options](#overwriting-options)
 
+## options.boundTestRun
+
+**Type**: Object
+
+If you need to call a selector from a Node.js callback, assign the current
+[test controller](../test-code-structure.md#test-controller) to the `boundTestRun` option.
+
+For details, see [Calling Selectors from Node.js Callbacks](selectors.md#calling-selectors-from-nodejs-callbacks).
+
+## options.timeout
+
+**Type**: Number
+
+The amount of time, in milliseconds, allowed for an element returned by the selector to appear in the DOM before the test fails.
+
+If the [visibilityCheck](#optionsvisibilitycheck) option is enabled, the element then must become visible within the `timeout`.
+
+**Default value**: timeout specified by using the [runner.run](../../using-testcafe/programming-interface/runner.md#run) API method
+or the [selector-timeout](../../using-testcafe/command-line-interface.md#--selector-timeout-ms) command line option.
+
+## options.visibilityCheck
+
+**Type**: Boolean
+
+`true` to additionally require the returned element to become visible within [options.timeout](#optionstimeout).
+
+**Default value**: `false`
+
 ## options.dependencies
+> Important! ***deprecated*** *Will be removed in next releases. Use [hierarchical selectors](selectors.md#find-elements-by-dom-hierarchy).*
```

Comment:
***Deprecated*** *Use [hierarchical selectors](selectors.md#find-elements-by-dom-hierarchy). The `option.dependencies` will be removed in future releases.*